<launch>
	<arg name="fps" default="30"/>

	<node pkg="track8yolo" type="imagepub" name="yolorespub" output="screen">	
		<param name="engine_path" value="$(find track8yolo)/weights/best_downward.engine" type="string"/>
		<param name="widthnet" value="640" type="int"/>
		<param name="heightnet" value="640" type="int"/>
		<param name="fps" value="$(arg fps)" type="int"/>
	</node>

	
	<node pkg="track8yolo" type="tracknode" name="uav_track" output="screen">	
		<param name="height" value="7.5" type="double"/>
		<param name="hz_control" value="$(arg fps)" type="int"/>
		<param name="land_speed" value="0.2" type="double"/>
		<param name="xkp" value="0.05" type="double"/>
		<param name="xkd" value="0.0" type="double"/>
		<param name="xki" value="0.0001" type="double"/>
		<param name="ykp" value="0.05" type="double"/>
		<param name="ykd" value="0.0" type="double"/>
		<param name="yki" value="0.0001" type="double"/>
	</node>

	<node pkg="rqt_reconfigure" type="rqt_reconfigure" name="rqt_reconfigure" output="screen" />

</launch>
